ext/on-modify.timewarrior

```diff
@@ -30,6 +30,9 @@ import json
 import subprocess
 import shlex
 
+# For issue #216 and generally sounds like a good idea
```

Comment:
Please remove this comment (If you like, put it into the commit message)

Reply:
Removed.